scripts/sexbound/plugins/pregnant/pregnant.lua

```diff
@@ -40,10 +40,58 @@ function Sexbound.Actor.Pregnant:onMessage(message)
     end
 end
 
+--- Returns a reference to this actor's insemination decay rate.
+function Sexbound.Actor.Pregnant:getInseminationDecay()
```

Comment:
This should be configurable too.

Reply:
The only two things you added to the config are the multiplier and max value. Nothing regarding the decay rate. So you'll have to include that too.